src/modules/store/plugins/vuex-url-search-params.js

```diff
@@ -0,0 +1,225 @@
+import _ from 'lodash'
+
+import { isNumber } from "@/numberUtils";
+
+// Persist and rehydrate the `Vuex` store via url search params (window.location.search)
```

Comment:
a more extensive comment/doc about how this persistance / rehydration works (conceptaully) would be helpful (since it's a central part of the app).